proto/metadata.proto

```diff
@@ -398,6 +398,16 @@ message Metadata {
     // Devices does not support setting prefix limit exceeded flag.
     // Juniper : b/317181227
     bool prefix_limit_exceeded_telemetry_unsupported = 139;
+    // Skip gNMI container OP tc.
+    bool skip_container_op = 140;
```

Comment:
```suggestion
   // Skip gNMI container OP tc.
   // Cisco: https://partnerissuetracker.corp.google.com/issues/322291556
    bool skip_container_op = 140;
```

Reply:
Please add sim. comments for all newly added deviations in the proto/metadata.proto file